graph/graph.go

```diff
@@ -98,18 +100,31 @@ func (g *Graph) loadFile(path string) (*build.File, error) {
 	return build.ParseBuild(validFilename, nil)
 }
 
-func (g *Graph) FormatFiles(write bool, out io.Writer) error {
+func (g *Graph) FormatFiles(write bool, out io.Writer, paths []string) error {
 	if err := g.ensureVisibilities(); err != nil {
 		return err
 	}
 	for _, file := range g.files {
+		if len(paths) != 0 && !g.isInScope(file.Path, paths) {
+			continue
+		}
 		if err := saveAndFormatBuildFile(file, write, out); err != nil {
 			return err
 		}
```

Comment:
I think we still want to save the file e.g. if it has a visibility change, but when it's out of scope, we want to run
```
build.FormatWithoutRewriting(buildFile)
```
instead of 
```
build.Format(buildFile)
```